Jenkinsfile

```diff
@@ -26,8 +26,8 @@ pipeline {
         VENV_DIRNAME = ".venv"
         BUILD_DIRNAME = "dist"
         RTOS_TEST_RIG_TARGET = "XCORE-AI-EXPLORER"
-        LOCAL_WIFI_SSID = credentials('bristol-office-test-ssid')
-        LOCAL_WIFI_PASS = credentials('bristol-office-test-wifi-password')
+        LOCAL_WIFI_SSID = credentials('bristol-office-development-wifi-ssid')
+        LOCAL_WIFI_PASS = credentials('bristol-office-development-wifi-password')
```

Comment:
Just to be sure, since this repository is publicly visible, does it pose a security vulnerability to expose the name of these credentials in this file?

Reply:
Essentially no, since these are pointers to credentials held in our private Jenkins credential store, it's possible to perhaps argue that it provides minimal fingerprinting information in our naming convention, but that's very tenuous. If an attacker could start attempting to get credentials from the Jenkins store, then we have much much larger problem already, and they will not be trying to steal the wifi ssid and password since they are already on the network.

Reply:
Thanks for your answer, @cs-xm.